common/server_room.cpp

```diff
@@ -36,6 +36,37 @@ Server_Room::~Server_Room()
     usersLock.unlock();
 }
 
+bool Server_Room::userMayJoin(const ServerInfo_User & userInfo)
+{
+    qDebug() << "USERPRIV: " << QString::fromStdString(userInfo.privlevel());
+
+    ServerInfo_Room roomInfo;
```

Comment:
You wouldn't call get..Display here, but this line doesn't need to be here, you're right